components/geometry-merger/index.js

```diff
@@ -2,81 +2,89 @@ if (!THREE.BufferGeometryUtils) {
   require('./lib/BufferGeometryUtils');
 }
 
-AFRAME.registerComponent('geometry-merger', {
-  schema: {
-    preserveOriginal: {default: false},
-    materialColors: {default: true}
-  },
-
-  init: function () {
-    var faceIndexEnd;
-    var faceIndexStart;
-    var self = this;
-
-    this.geometry = new THREE.Geometry();
-    this.mesh = new THREE.Mesh(this.geometry);
-    this.el.setObject3D('mesh', this.mesh);
-
-    this.faceIndex = {};  // Keep index of original entity UUID to new face array.
-    this.vertexIndex = {};  // Keep index of original entity UUID to vertex array.
+AFRAME.utils.setBufferGeometryColor = function () {
```

Comment:
This is a useful utility function to have (the basic example makes use of it). I think it makes sense in AFRAME.utils, rather than being a method on a the geometry-merger component, as it's also useful in cases where geometry-merger is not used (for coloring vertices on any BufferGeometry).

I don't know whether there is any established precedent for extending AFRAME.utils when including a community component, but I don't see a better way of making this available.

(other option might be to extend THREE.BufferGeometryUtils rather than AFRAME.utils???)